internal/daemon/worker/worker.go

```diff
@@ -255,6 +256,7 @@ func (w *Worker) Shutdown() error {
 	w.baseCancel()
 	w.Resolver().UpdateState(resolver.State{Addresses: []resolver.Address{}})
 
+	w.NodeeFileStorage.Cleanup()
```

Comment:
Do we want to clear out worker storage on shutdown? If a user is taking a worker down for maintenance, I don't think we'd want a worker's creds cleared out

Reply:
I took this away for now. Perhaps we want to look for a specific worker tag before calling this function